packages/app/components/Queue/QueueListSharedComponents.tsx

```diff
@@ -142,6 +182,14 @@ export function QueueInfoColumn({
       {buttons}
       <StaffH2>Staff</StaffH2>
       <TAStatuses queueId={queueId} />
+      {isTA && (
+        <DisableQueueButton
+          onClick={confirmDisable}
+          data-cy="queue-disable-button"
+        >
+          Disable Queue
+        </DisableQueueButton>
+      )}
```

Comment:
This button gets bumped up right under the TA status cards on my smaller laptop screen - any way you guys could get them to space out more? (looks like setting the InfoColumnContainer to have a flex column display + giving the TAStatuses a bottom margin should work)

Reply:
like this?
![image](https://user-images.githubusercontent.com/23366632/146847071-e6830f38-4f3e-4df0-89b4-451b8a751e5a.png)